packages/jest-cli/src/SearchSource.js

```diff
@@ -76,6 +77,15 @@ const toTests = (context, tests) =>
     path,
   }));
 
+const fileExists = (filePath: string) => {
```

Comment:
we already have like 3 functions like this in the codebase. It sucks that we can't just share it :(